lib/system.nim

```diff
@@ -2596,6 +2596,9 @@ when not defined(JS): #and not defined(nimscript):
     proc endOfFile*(f: File): bool {.tags: [], benign.}
       ## Returns true iff `f` is at the end.
 
+    proc fileError*(f: File): bool {.tags: [], benign.}
```

Comment:
Is there a reason for it? I thought if users want to test for the eor they might want to test the error flag as well. Or is it replaced by exceptions?